datumaro/plugins/coco_format/extractor.py

```diff
@@ -259,10 +315,16 @@ def _load_annotations(self, ann, image_info=None, parsed_annotations=None):
         ):
             label_id = self._get_label_id(ann)
 
-            attributes["is_crowd"] = bool(ann["iscrowd"])
+            attributes["is_crowd"] = bool(self._parse_field(ann, "iscrowd", int))
 
             if self._task is CocoTask.person_keypoints:
-                keypoints = ann["keypoints"]
+                keypoints = self._parse_field(ann, "keypoints", list)
+                if len(keypoints) % 3 != 0:
+                    raise InvalidAnnotationError(
+                        "Keypoints have invalid value count %s, which is not divisible by 3. "
+                        "Expected (x, y, visibility) triplets." % (len(keypoints),)
```

Comment:
You seem to be using f-strings everywhere, so why not here?

Reply:
I don't have preference to any of these styles.